src/Core/System/User/Command/UserChangePasswordCommand.php

```diff
@@ -44,21 +43,28 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $username = $input->getArgument('username');
         $password = $input->getOption('password');
 
-        $userId = $this->getUserId($username, $context);
-        if ($userId === null) {
-            $io->error(sprintf('The user "%s" does not exist.', $username));
-
-            return self::FAILURE;
-        }
+        if (!$password) {
+            $passwordQuestion = new Question('Enter new password for user');
+            $passwordQuestion->setValidator(static function ($value): string {
+                if ($value === null || trim($value) === '') {
+                    throw new \RuntimeException('The password cannot be empty');
+                }
 
-        if (empty($password)) {
-            $passwordQuestion = new Question('New password for the user');
+                return $value;
+            });
```

Comment:
this too cannot be empty, because it is requiered

Reply:
Please take a look here https://streamable.com/256gh0

Without this check for an empty password, `setMaxAttempts` is not doing it's intended purpose.